airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py

```diff
@@ -143,6 +144,13 @@ def test_check(response, start_date, check_passed):
 @pytest.mark.parametrize(
     "ticket_forms_response, status_code, expected_n_streams, expected_warnings",
     [
+        ({"ticket_forms": [{"id": 1, "updated_at": "2021-07-08T00:05:45Z"}]}, 200, 20, []),
+        (
+            {"error": "Not sufficient permissions"},
+            403,
+            19,
+            ["Skipping stream ticket_forms: Check permissions, error message: Not sufficient permissions."],
+        ),
```

Comment:
Just for my understanding, why with identical  ticket_forms_response and status_code we receive different number of streams? 20 and 27 for 200 status code, 19 and 24 for 403 status code